delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/cache/LocalCache.scala

```diff
@@ -0,0 +1,293 @@
+package ch.epfl.bluebrain.nexus.delta.kernel.cache
+
+import cats.effect.IO
+import com.github.benmanes.caffeine.cache.{Cache, Caffeine}
+
+import scala.concurrent.duration._
+import scala.jdk.CollectionConverters._
+import scala.jdk.DurationConverters._
+
+/**
+  * An arbitrary key value store.
+  *
+  * @tparam K
+  *   the key type
+  * @tparam V
+  *   the value type
+  */
+trait LocalCache[K, V] {
```

Comment:
Equivalent of `KeyValueStore` but for CE.
Renamed as `KeyValueStore` name was some left over the period where it has a Caffeine implementation but also one relying on a distributed Akka cache